csrc/mmdeploy/net/rknn/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+project(mmdeploy_rknn_net)
+
+add_library(rknn SHARED IMPORTED)
+
+if (NOT EXISTS $ENV{RKNN_ROOT}/runtime/RK3588/Linux/librknn_api/aarch64/)
```

Comment:
May put them in a toolchain cmake file like `cmake/toolchains/riscv64-unknown-linux-gun.cmake` did.

Reply:
Done